metrics/deploy/docs/diagrams/deploy_sequence.puml

```diff
@@ -0,0 +1,50 @@
+@startuml deploy_sequence
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant Terminal
+
+box CLI
+participant DeployCommand
+participant FirebaseCommand
+participant FlutterCommand
+participant GCloudCommand
+endbox
+
+Bob -> Terminal : enters 'build/metrics deploy'
+
+activate Terminal
+Terminal -> DeployCommand : run()
+
+activate DeployCommand
+
+DeployCommand -> GCloudCommand : login()
+DeployCommand -> GCloudCommand : addProject()
+activate GCloudCommand
+GCloudCommand --> DeployCommand : project ID
+deactivate GCloudCommand
```

Comment:
If a participant returns and then becomes deactivated then use either `return` or `--` shortcut. For example here:
```suggestion
DeployCommand -> GCloudCommand ++ : addProject()
GCloudCommand --> DeployCommand -- : project ID
```

But in this case, this is better: 
```suggestion
DeployCommand -> GCloudCommand ++ : addProject()
return project ID
```

However, it seems like `GCloudCommand` should be activated earlier.